payments/models.py

```diff
@@ -105,46 +81,95 @@ class Meta:
         abstract = True
 
 
-class BerthProduct(AbstractPlaceProduct):
-    price_group = models.ForeignKey(
-        BerthPriceGroup,
-        on_delete=models.CASCADE,
-        verbose_name=_("price group"),
-        related_name="products",
+class BerthProductManager(models.Manager):
+    def get_in_range(self, width: Union[Decimal, float, int]):
+        return (
+            self.get_queryset()
+            .filter(min_width__lt=width, max_width__gte=width)
```

Comment:
```suggestion
            .filter(min_width__lte=width, max_width__gt=width)
```

Should the equality check be moved to the minimum width instead?

Reply:
Judging by the table I think the way it's implemented is the intention. The implementation doesn't cover the case where `width == 0` though

Reply:
It was on purpose to have the `min < width <= max` 

**Original:**
| Min | Max |
| --- | --- |
| 0.0 | 2.0 |
| 2.1 | 2.75 |
| 2.76 | 3.0 |

**_Our_ version:**
| Min | Max |
| --- | --- |
| 0.0 | 2.0 |
| 2.0 | 2.75 |
| 2.75 | 3.0 |

So, to be able to keep the same values on the same ranges, we need to do it like that.
```
--|(---------]|(-------]|(------]|
  0           2        2.75      3
```